README.md

```diff
@@ -1 +1,155 @@
-# cryptollar
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+<img src="./src/assets/cryptollar.png" alt="cryptollar logo" width="140"  height="auto" />
+  <br/>
+  <h1><b>CRYPTOLLAR</b></h1>
+</div>
+
+</div>
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Screenshots](#screenshots)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 CRYPTOLLAR<a name="about-project"></a>
+
+CRYPTOLLAR - is a simple website for presenting prices of cryptocurrencies. It is part of Microverserse software training challeneges; a capstone project demonstrating use of React and Redux packages for frontend development.
+
+## 🛠 Screenshots <a name="screenshots"></a>
+
+<img src="./src/assets/crytollar_screenshot-d.JPG" width="70%" alt="desktop screenshot" />
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- **React**
+- **Redux**
+- **Html 5**
+- **CSS 3**
+
+### Key Features <a name="key-features"></a>
+
+- Listing of cryptocurrencies daily quotes.
+- Detail page of any selected crypto.
+- Multi-page React routing.
+- Use of Redux thunk for async operations.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- Here is the link to the live demo version - [cryptollar](https://cryptollar.onrender.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites <a name="prerequisites">
+
+In order to run this project you need:
+
+### Setup <a name="setup">
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-project-folder
+  git clone https://github.com/adamilare/cryptollar.git
+```
+
+### Install <a name="install">
+
+Install this project with:
+
+Example command:
+
+```sh
+  cd cryptollar
+  npm install
+```
+
+-
+
+### Usage <a name="usage">
+
+To run the project, starting the React App, execute the following command:
+
+```sh
+  npm start
+```
+
```

Comment:
- [ ] [OPTIONAL]
It will be a good idea to add instructions on how to run test cases for your app, in the _Getting Started_ section